services/core-api/app/api/projects/namespace.py

```diff
@@ -1,6 +1,6 @@
 from flask_restplus import Namespace
 
-from app.api.projects.project.resources.project import ProjectResource, ProjectListResource
+from app.api.projects.project.resources.project import ProjectResource, ProjectListResource, MajorProjectListDashboardResource
```

Comment:
should the name of the new resource be inline with the others? ProjectListDashboardResource?